Dockerfile

```diff
@@ -18,6 +18,7 @@ ONBUILD ARG DEPTH_MERGE=100
 ONBUILD ARG CLEAN=true
 ONBUILD ARG COMPILE=true
 ONBUILD ARG CONFIG_BUILD=true
+ONBUILD ARG AUTO_REQUIREMENTS=true
```

Comment:
This is more for a development box, and while it can cause issues it also is incredibly helpful (at least is has been for me). 

Is there any reason to not leave this feature as is, just with the good default? This is one of the last things I need to move off of my fork.